docs/docs/about/why-cli.mdx

```diff
@@ -2,8 +2,60 @@
+- ***[Manage SharePoint Framework solutions](../cmd/spfx/spfx-doctor)***. CLI for Microsoft 365 will help you setup your local environment for SPFx development and will help you with common tasks when developing SPFx solutions, like validating its correctness or getting guidance on how to upgrade it.
+- Make [CLI for Microsoft 365 your own](../user-guide/configuring-cli). CLI for Microsoft 365 offers a variety of settings that will allow you to customize it for your specific use case or needs. If going over the settings is challenging, then you will love the [m365 setup](../cmd/setup) command that will help you customize CLI for common use cases like setting it to be tailored to be run in a script.
+- AI, Of course, we have an AI! CLI for Microsoft 365 comes along with an [AI assistant chili 🌶️](../user-guide/chili) that will help you find the right commands for your tasks.
+- How many CLI's allows you to run it as your API in an app? That is what I though. [CLI for Microsoft 365 may be used as an API](../user-guide/use-cli-api) in your Node.js app.
+- You may filter and sort the output of every command. [Filtering and sorting](../user-guide/filter-cli-data.mdx) is a powerful feature that allows you to get only the data you need and in the order you want.
+
+Interested in more? Start with our [user guide](../user-guide/using-cli.mdx) and learn how to use the CLI for Microsoft 365.
+
+## Why not another approach or a different CLI?
+
+### Why not just do things manually?
+
+Using CLI for Microsoft 365 is not about replacing the UI, it's about making your life easier. If you need to do a repetitive task, like creating a new SharePoint list every time a new project is started, you can create a script and automate this task with CLI for Microsoft 365. This way you can save time and avoid human errors.
+
+### Why not just write your own CLI or some custom console app tailored to your needs?
+
+You could, but why reinvent the wheel? CLI for Microsoft 365 is a powerful tool that is being actively developed and maintained by a team of experts. It's open-source and free to use. It's also cross-platform and works on any device. It's a great tool that can help you manage your Microsoft 365 tenant more efficiently.
+
+### Why not use SharePoint Online Management Shell?
+
+SharePoint Online Management Shell is a great tool, but it's limited to SharePoint Online and only works with Windows PowerShell. Additionally, it only supports SharePoint administration tasks, so you can only use it to run admin-related commands. In contrast, CLI for Microsoft 365 offers a variety of commands from many different areas of Microsoft 365 like SharePoint Online, Teams, Viva Engage, Entra, Power Apps, Power Automate, Power Platform, OneDrive, Planner, OneNote, Outlook, SharePoint Embedded, and many more. It's a more comprehensive tool that can help you manage your Microsoft 365 tenant more efficiently.
```

Comment:
I wouldn't repeat all the areas again. We already did that at the start of this page.